sdk/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License

```

Comment:
I think the License file is uneccessary, as it is just a duplicate of the top-level license. On the other hand, it also doesn't hurt, so I don't know what we should do here.

Reply:
There was a problem with building source distribution when the pyproject.toml file was referencing to a LICENSE file not within /sdk
I have not found a solution as I only found an issue covering this but there was no fix proposed.

Reply:
Okay, in that case, it's also fine to leave it in. It's not like this file is likely to change (other than the year at the top).